composer.json

```diff
@@ -16,15 +16,15 @@
     ],
     "require": {
         "php": ">=7.4",
-        "gacela-project/gacela": "dev-master"
+        "gacela-project/gacela": "0.10.0"
```

Comment:
Should we require the `dev-master´ version or a fixed one @Chemaclass ?

Reply:
I would prefer to depend on a fixed version in this repository, because it's not another "example" repository, but a real one that needs to work all the time.